README.md

```diff
@@ -358,7 +358,7 @@ impl serde::Serialize for i32 {
 As you can see it's pretty simple. More complex types like `BTreeMap` need to
 pass a
 [MapVisitor](http://serde-rs.github.io/serde/serde/serde/ser/trait.MapVisitor.html)
-to the 
```

Comment:
please leave those spaces in there, I know they are unfortunate, but otherwise the next line is very long.